tests/integration/test_cos.py

```diff
@@ -0,0 +1,166 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Integration tests for jenkins-k8s-operator with COS."""
+
+import functools
+import typing
+
+import pytest
+import requests
+from juju.action import Action
+from juju.application import Application
+from juju.client._definitions import FullStatus
+from juju.model import Model
+from kubernetes.client import CoreV1Api
+
+from .helpers import wait_for
+from .types_ import UnitWebClient
+
+
+@pytest.mark.abort_on_fail
+async def test_prometheus_integration(
+    unit_web_client: UnitWebClient, prometheus_related: Application
+):
+    """
+    arrange: none.
+    act: deploy the WordPress charm and relations established with prometheus.
+    assert: prometheus metrics endpoint for prometheus is active and prometheus has active scrape
+        targets.
```

Comment:
Isn't the arrange to setup the charm and relate to prometheus? Arrange is usually the setup procedure of the test like deploying something or relating something

Reply:
Fixed! Thank you!